nx/lib/nx.ex

```diff
@@ -6408,15 +6408,160 @@ defmodule Nx do
 
   """
   @doc type: :ndim
-  def dot(t1, axes1, t2, axes2) do
+  def dot(t1, axes1, t2, axes2), do: dot(t1, axes1, [], t2, axes2, [])
+
+  @doc """
+  Computes the dot-product of two batched tensors, throughout the batch dimensions.
+
+  ## Examples
+
+    ### Dot product between two batched tensors
```

Comment:
```suggestion
  ### Dot product between two batched tensors
```

Reply:
Some day I'll remember which is the correct indentation :laughing: